x-pack/plugins/task_manager/server/task_claimers/strategy_mget.ts

```diff
@@ -320,10 +323,22 @@ async function searchAvailableTasks({
     definitions,
   });
   const partitions = await taskPartitioner.getPartitions();
-  if (partitions.length === 0) {
+  if (
```

Comment:
I think the combination of the two `if` statements below will help reduce the noise. It will start logging on the first time no partitions were found and won't log again for another minute. It'll then log a message once partitions are found if a previous warning message was logged.

The only flaw is if there's flapping between the two, we'll see alternating messages between partitions not found and found. But perhaps it won't happen often and we can keep this code simple..